.github/actions/build/action.yml

```diff
@@ -41,6 +41,31 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Parse pubspec.yaml for version
```

Comment:
move out check-version from build action into the new action

Reply:
done

Reply:
android and iOS start building

Reply:
i just added it to Release Tag after checked and confirmed by you we can add to other files